src/main/service/listParseHelperService.ts

```diff
@@ -0,0 +1,359 @@
+import {partyRoleMappings} from '../models/consts';
+import moment from 'moment-timezone';
+
+export class ListParseHelperService {
+
+  public timeZone = 'Europe/London';
+
+  /**
+   * Manipulate the daily cause list json data for writing out on screen.
+   * @param dailyCauseList
+   */
+  public manipulatedDailyListData(dailyCauseList: string): object {
```

Comment:
This needs to be renamed to somethibg more generic.....I see it is used by public list and standard list too, so if possible don't use the word 'DailyLIst' in the method name